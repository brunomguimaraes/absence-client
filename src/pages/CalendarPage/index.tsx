
import React from 'react';
import CalendarComponent from '../../components/CalendarComponent';

const CalendarPage = () => {
    return (
        <div id="page-calendar">
          <CalendarComponent />
        </div>
    );
};

export default CalendarPage;


