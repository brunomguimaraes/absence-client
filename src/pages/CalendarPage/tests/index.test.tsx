import React from 'react';
import { shallow } from 'enzyme';

import CalendarPage from '..';

describe('CalendarPage', () => {
  it('renders the component', () => {
    const component = shallow(<CalendarPage />);
    expect(component).toMatchSnapshot();
  });
});
