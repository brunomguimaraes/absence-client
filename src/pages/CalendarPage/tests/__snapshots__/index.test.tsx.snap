// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarPage renders the component 1`] = `ShallowWrapper {}`;
